crates/astria-sequencer/src/service/info/mod.rs

```diff
@@ -64,6 +64,12 @@ impl Info {
                 crate::asset::query::allowed_fee_asset_ids_request,
             )
             .context("invalid path: `asset/allowed_fee_asset_ids`")?;
+        query_router
+            .insert(
+                "bridge/account_last_tx_hash/:address",
```

Comment:
Not blocking here, because this matches existing pattern but I'd prefer to have resource name style path based queries is `/resource-type/{id}/sub-resource-type/{id}`. In this instance that would be `bridge/{address}/last_tx_hash` so it's clear who owns what property hierarchically. The `last_tx_hash` is a property of the address, which is a bridge.

This would apply to all our queries though.